presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClientModule.java

```diff
@@ -40,13 +39,11 @@ protected void setup(Binder binder)
     private static void ensureCatalogIsEmpty(String connectionUrl)
     {
         try {
-            Driver driver = new Driver();
-            Properties urlProperties = driver.parseURL(connectionUrl, null);
-            checkArgument(urlProperties != null, "Invalid JDBC URL for MySQL connector");
-            checkArgument(driver.database(urlProperties) == null, "Database (catalog) must not be specified in JDBC URL for MySQL connector");
+            ConnectionUrlParser parser = parseConnectionString(connectionUrl);
+            boolean nullOrEmpty = isNullOrEmpty(parser.getPath());
         }
-        catch (SQLException e) {
-            throw new RuntimeException(e);
+        catch (CJException ignore) {
+            boolean nullOrEmpty = false;
```

Comment:
Declare `nullOrEmpty` variable as method variable on top once.
In which scenario `CJException` would come?

Reply:
@Akanksha-kedia